### ISSUE TITLE
Date de publication et de modification des articles

### ISSUE THREAD

**Issue by Jireille** [lovasoa/ophirofox#239]
Bonjour,
Avec un exemple un article de Libération, j'ai compris que lorsque la réponse à une recherche est "aucun document disponible pour la période sélectionnée", c'est que la date prise comme date de publication de l'article est la date de modification. 
https://www.liberation.fr/international/europe/commission-europeenne-le-patriarcat-a-de-beaux-jours-devant-lui-a-bruxelles-20240827_OEFBV3GXHRBSFFLARHSVKUSBME/
publié le 27 août 2024 à 20h18
(mis à jour le 30 août 2024 à 12h40)
La recherche est faite par Ophirofox sur la date du 30 et pas celle du 27. 
Je ne sais pas si tous les journaux distinguent correctement, d'une façon facile à coder, les dates de publication/modification.
Merci de creuser au cas où une modification est possible.

**Comment by Write** [comment]
Ils modifient <meta property="date:published_time" content="2024-08-30"/> à la date de dernière modification, ce qui est de mon avis, une erreur de leur part. 

Je ne trouve pas de balise meta avec la date de modification.

**Comment by Jireille** [comment]
Merci ! Ce n'est pas très pro dans les journaux de ne pas mettre de balise de date de modification. Pour le moment l'important est d'avoir compris pourquoi il n'y a pas toujours de réponse pour les articles modifiés. Dans ce cas, je ferai une recherche manuelle en élargissant la période de recherche. Bon dimanche.

**Comment by lnoss** [comment]
C'est moi qui ai intégré la recherche approximative bornée par la date de publication d'un article (#192) pour réduire le temps de recherche et un peu de grammes de carbon. 

J'ai aussi eu le loisir de constater au fil de l'année que ce n'était pas assez rigoureux pour certains médias (de mémoire pas uniquement Libération). Je n'ai pas eu le temps de m'y pencher, car il faut tomber sur la perle rare et c'est reproduisible pendant moins de 24 heures.

Franchement, ils sont relous de s’asseoir sur le protocole Open Graph. J'ai déjà vu des articles modifiés des années plus tard alors si d'autres médias pratiquent la même mauvaise habitude... Je vois peu d'options si ce n'est laisser la possibilité pour le parseur de chaque média de retourner spécifiquement une date (ou envoyer un courriel au Libé). 

Bref, ils sont relous.

**Comment by Write** [comment]
Les modifications apportées par @lnoss ayant été intégrées, cela a-t-il pu permettre de résoudre le problème ?

**Comment by Jireille** [comment]
Oui, je viens de faire la recherche pour ce même article en bornant du 27 au 29 août (donc en excluant la date de correction) et j'ai eu la réponse !  J'ai fini par trouver un autre exemple https://www.liberation.fr/idees-et-debats/tribunes/union-europeenne-laccord-mercosur-doit-tomber-20241122_E5K32QJ5O5CXFBL7SMRL2PZ6GM/
qui donne une réponse OK (la date de création est le 22/11 et la date de modification le 27/11).
 https://nouveau-europresse-com.bnf.idm.oclc.org/Search/ResultMobile
Merci !

**Comment by Write** [comment]
Super, merci pour la précision, je clos donc ce ticket et merci encore à Inoss pour le fixe.
